dev-util/intel-graphics-compiler/intel-graphics-compiler-1.0.3445.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_ECLASS=cmake
+inherit cmake-multilib llvm
```

Comment:
Please reinstate LLVM_MAX_SLOT. The chances of IGC not being updated to support a newer version of LLVM when opencl-clang is are very nearly zero but it doesn't change the fact IGC does not support _any_ version of LLVM...

Reply:
reinstated. I checked that lowering it to 8 pulls in the dependencies correctly. at least for `emerge -p`